src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  die() {
+    Animal.alive = Animal.alive.filter(animal => animal.health > 0);
```

Comment:
```suggestion
    Animal.alive = Animal.alive.filter(animal => animal.health >= 0);
```

Reply:
Herbivores should be removed from 'Animal.alive' array,                                              
  when their health is 0.